### DISCUSSION TITLE
The graphic is abnormal and all text on the screen cannot be displayed for all render

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#30722] [Q&A] [answered]
Question submitted on behalf of my friend

[DxDiag.txt](https://github.com/user-attachments/files/17791113/DxDiag.txt)
[logs(1).zip](https://github.com/user-attachments/files/17791118/logs.1.zip)

tried all render but still

also tired re-install
ensured the latest graphic driver version 

![27f741a54246f48bd9e49bd13869b2a4](https://github.com/user-attachments/assets/c67248c5-5002-453b-9262-5cd7a17898d4)
![22e3b936842fb21e043449464622c2e8](https://github.com/user-attachments/assets/d38a1846-fed4-4f2e-a0bb-799c02826846)
![ba5d72cca2a5cd3c4449fcc4a56e3d53](https://github.com/user-attachments/assets/6eaf8938-c077-4865-a6b8-cada6bfe0ebc)

**Comment by Paladynee** [comment]
it would probably be more beneficial to open an issue instead

**Reply by cdwcgt** [reply]
Because I'm not sure if it's my friend's system problem or a real bug in lazer, I chose to post it here

**Comment by Joehuu** [comment, marked as answer]
Maybe try what's said in https://github.com/ppy/osu/issues/14755#issuecomment-919870382. If it fixes, zip and send the problematic folder.

**Reply by cdwcgt** [reply]
yes this was fixed the problem.
here is the archive

[fonts.zip](https://github.com/user-attachments/files/17807989/fonts.zip)
